app/assets/javascripts/govuk_publishing_components/components/feedback.js

```diff
@@ -90,6 +98,14 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
         that.$somethingIsWrongForm.append($('<input type="hidden" name="referrer">').val(document.referrer || 'unknown'))
       }
 
+      function setHiddenValuesNotUsefulForm () {
+        var currentPathName = window.location.pathname.replace(/[^\s=?&]+(?:@|%40)[^\s=?&]+/, '[email]')
+        var finalPathName = encodeURI(currentPathName)
+
+        that.$surveyForm.append($('<input name="email_survey_signup[ga_client_id]" type="hidden">').val(window.GOVUK.cookie('_ga') || '0'))
+        that.$surveyForm.append('<a href="https://www.smartsurvey.co.uk/s/gov-uk-banner/?c=' + finalPathName + '&amp;gcl=' + window.GOVUK.cookie('_ga') + 'class="gem-c-feedback__email-link govuk-link" id="take-survey" target="_blank" rel="noopener noreferrer">Don’t have an email address?</a>')
```

Comment:
Same comment as above - I'd prefer to fetch the value of the cookie and assign that to a variable we use here, rather than setting the value of the cookie and reading from that again.